### PR TITLE
"Copyedit "Transatlantic."

### PR DESCRIPTION
**Pull request by sarahmonster**
You abuse colons.

### CODE REVIEW

File: _posts/2015-01-12-transatlantic.md [hunk 1]
```diff
@@ -9,78 +9,76 @@ credit: Charlie Marshall
...
+The UK’s Visa and Immigration Office is unclear on the rules for British citizens self-employed outside the UK. Nowhere were the rules for this situation explained in a straightforward way. Most information assumes that your sponsor is living in the UK and employed in the UK. If that isn't the case, you're very much on your own.
 
 [Sarah]: http://triggersandsparks.com/
 
 ## The UK and immigration
 
-Sentiment around immigration in the UK is like that in the USA (see: heated). They are a wealthy country near many poorer countries. And unlike the borders between USA and Mexico, **borders between poorer EU countries and the UK are _open_**. Illegal immigration is a problem in the UK, but many consider *legal* EU immigration as big of a problem.
+Sentiment around immigration in the UK is similat to in the USA (see: heated). It's a wealthy country near many poorer countries. And unlike the borders between USA and Mexico, **borders between have-not EU countries and the UK are _open_**. Illegal immigration is a problem in the UK, but many consider *legal* EU immigration as big a problem.
```
Comment (tofumatt) [inline _posts/2015-01-12-transatlantic.md:24]: "similat" sounds like an alien or a weird smoothie powder,

File: _posts/2015-01-12-transatlantic.md [hunk 2]
```diff
@@ -9,78 +9,76 @@ credit: Charlie Marshall
...
 
-You can find plenty of stories about a United Kingdom flooded by immigrants or about a London where no one speaks English anymore. _These views are sensationalist to say the least._ Open borders with poorer, Eastern European countries is supplying cheap labour--both skilled and unskilled. **The fight against immigration is one that wins votes, but rarely makes economic sense.** (Unless your current market is so fucked you need massive protectionism.)
+You can find plenty of stories about a United Kingdom flooded by immigrants or about a London where no one speaks English anymore. (And there's always Birmingham, ostensibly a town where non-Muslims are no longer allowed.)  _These views are sensationalist at best, blatantly ignorant at worst._ Open borders with other countries supplys cheap labour--both skilled and unskilled.
```
Comment (tofumatt) [inline _posts/2015-01-12-transatlantic.md:26]: Not using Fox News. :wink:

File: _posts/2015-01-12-transatlantic.md [hunk 3]
```diff
@@ -9,78 +9,76 @@ credit: Charlie Marshall
...
 
 ### Restricting British families
 
 In 2012 UK Parliament updated immigration rules for British citizens with non-EU spouses. British citizens [must make at least £18,600/year to sponsor their partners](http://www.theguardian.com/law/2014/jul/11/appeal-court-18600-foreign-spouse-uk). This was--and still is--a controversial rule. On minimum wage, a British citizen would not earn enough to sponsor their spouse. Earning a decent wage in a poorer country excludes UK expats from returning with their families.
 
 <span id="r-1"></span>
-I am not a fan of this rule. Fortunately, my girlfriend makes enough to qualify us. We're great immigrants<sup>[1](#footnote-1)</sup> as we're both creating new jobs in the UK.
+I am not a fan of this rule. Fortunately, my girlfriend makes enough to qualify us. (What a great moneybags girl! You should marry the pants off her!) Together, we're a great immigration candidate<sup>[1](#footnote-1)</sup> as we're both creating new jobs in the UK.
```
Comment (tofumatt) [inline _posts/2015-01-12-transatlantic.md:35]: I'm starting to suspect your sister helped edit this.

File: _posts/2015-01-12-transatlantic.md (hunk 1 above)
Comment (sarahmonster) [inline _posts/2015-01-12-transatlantic.md:24]: Whatever, apostrophe.

File: _posts/2015-01-12-transatlantic.md (hunk 3 above)
Comment (sarahmonster) [inline _posts/2015-01-12-transatlantic.md:35]: 100% me. Although she did laugh at my colon comment. (I bought her cider.)

File: _posts/2015-01-12-transatlantic.md [hunk 4]
```diff
@@ -9,78 +9,76 @@ credit: Charlie Marshall
...
 
-But: if you're a British citizen working outside the UK, you'll need more documentation. A **lot** more.
+But if you're a British citizen working outside the UK, you'll need more documentation. A **lot** more.
 
 **It is possible to apply when both partners are outside the UK.** That wasn't even clear to me when I started. Here's how we did it:
 
 ### Self-Employment Evidence
 
-First thing's first: the self-employed sponsor should be able to "move" their business. _Without the intent to move their business, UK Immigration will refuse your application._ Proving this is as simple as writing it in a letter, but it's worth mentioning.
+First things first: the self-employed sponsor should be able to "move" their business. _Without the intent to move the business, UK Immigration will refuse your application._ Proving this is as simple as writing it in a letter, but it's worth mentioning.
```
Comment (tofumatt) [inline _posts/2015-01-12-transatlantic.md:47]: See, I will go with your suggestion, but "First thing is first" totally makes sense so I don't feel like my version is wrong. It's entirely possible it's not the turn of phrase, but that could just be me misappropriating the meaning in my head.

Comment (sarahmonster) [inline _posts/2015-01-12-transatlantic.md:47]: It isn't the phrase, and it doesn't make any sense. It's "things" plural. http://idioms.thefreedictionary.com/first+things+first

Also: http://english.stackexchange.com/questions/7419/which-is-the-correct-idiom-first-things-first-or-first-things-first

File: _posts/2015-01-12-transatlantic.md (hunk 3 above)
Comment (tofumatt) [inline _posts/2015-01-12-transatlantic.md:35]: I suspect you bought yourself some too, eh?

Comment (sarahmonster) [inline _posts/2015-01-12-transatlantic.md:35]: I'm the designated driver!

Comment (tofumatt) [inline _posts/2015-01-12-transatlantic.md:35]: Enjoy a tipsy pillion then! I can tell you from experience it's _so much fun_.

Comment (sarahmonster) [inline _posts/2015-01-12-transatlantic.md:35]: I'm in a Jetta now—I should be able to manage.

**Pull request merged**